lib/persisters/s3.js

```diff
@@ -25,7 +25,17 @@ class S3 {
    */
 
   load() {
-    return this.lambda.get(this.bucket, this.key)
+    return new Promise((resolve, reject) => {
+      this.lambda.get(this.bucket, this.key)
+        .then(resolve)
+        .catch((e) => {
+          if (String(e).indexOf('NoSuchKey') > -1) {
```

Comment:
oh I see, if it doesn't exist, use an empty string. maybe this can be an option for persist? might be weird getting an empty string back sometimes. depends on what you're storing I suppose